src/pages/Home/Home.js

```diff
@@ -1,34 +1,21 @@
 /* eslint-disable react/jsx-fragments */
-import React, { Fragment, useState, useContext } from 'react';
+import React, { useState, useContext } from 'react';
 import { Navbar, Toolbar, TorrentsList } from '../../containers';
 import StormContext from '../../context/Storm.context';
 import { HomePageContainer } from './Home.styles';
 
 const Home = () => {
   const [activeFilter, setActiveFilter] = useState('all');
-  const {
-    data:
-    { torrentSelected },
-    actions: { clearTorrentSelection },
-  } = useContext(StormContext);
+  const { actions: { clearTorrentSelection } } = useContext(StormContext);
+
   return (
-    <Fragment>
-      <Navbar
-        activeFilter={activeFilter}
-        setActiveFilter={setActiveFilter}
-      />
+    <>
+      <Navbar activeFilter={activeFilter} setActiveFilter={setActiveFilter} />
       <Toolbar setActiveFilter={setActiveFilter} />
-      <HomePageContainer onClick={() => {
-        if (torrentSelected) {
-          clearTorrentSelection();
-        }
-      }}
-      >
-        <TorrentsList
-          activeFilter={activeFilter}
-        />
+      <HomePageContainer onClick={() => clearTorrentSelection()}>
```

Comment:
would be cool if you can replace this with `<HomePageContainer onClick={clearTorrentSelection}>`